docs/user-guide/configuration/annotations.md

```diff
@@ -33,7 +33,7 @@ Some Annotations are used to modify the properties of the build environment. The
 | Annotation | Values | Description |
 |------------|--------|-------------|
 | beta.screwdriver.cd/executor | Ask your cluster admin | This will determine what compute system is used to run the build. For example, set the build to run in a VM, a kubernetes pod, a docker container, or a Jenkins agent. |
-| beta.screwdriver.cd/cpu | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 2 CPU resources (`LOW`) and 6 CPU resources (`HIGH`). Default is `LOW`. |
-| beta.screwdriver.cd/ram | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 2 GB RAM (`LOW`) and 12 GB RAM (`HIGH`). Default is `LOW`. |
+| beta.screwdriver.cd/cpu | `MICRO` / `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 0.5 (`MICRO`), 2 (`LOW`) and 6 (`HIGH`) CPU resources. Default is `LOW`. |
```

Comment:
I think "between" should be "among".

Reply:
I think both is OK, but between is better in this scene.
https://eikaiwa.weblio.jp/column/study/eigo-bunpo/between-among
http://nagatuduki-eikaiwa.com/9592.html
https://www.englishcafe.jp/answer/anz-1c.html

Reply:
🙆